### PR TITLE
add links to tutorial repos + code style + copyright fix

### PR DESCRIPTION
**Pull request by gzhao408**

### CODE REVIEW

File: src/tutorials/deploy-vyper.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Deploying Vyper Contracts on Oasis (Uniswap)
 
-[Uniswap](https://uniswap.io/) is a decentralized exchange (DEX) for ERC20 tokens built on Ethereum. Here, it is configured for deployment on Oasis. This tutorial's methods for deploying Solidity contracts using Truffle and Web3.js are applicable to any Ethereum contract written in Vyper. For contracts written in Solidity, see the Compound tutorial.
+[Uniswap](https://uniswap.io/) is a decentralized exchange (DEX) for ERC20 tokens built on Ethereum. Here, it is configured for deployment on Oasis. This tutorial's methods for deploying Vyper contracts using Truffle and Web3.js are applicable to any Ethereum contract written in Vyper. For contracts written in Solidity, see the [Compound tutorial](./deploy-solidity.md).
```
Comment (nhynes) [inline src/tutorials/deploy-vyper.md]: sentence per line, please

Comment (gzhao408) [reply src/tutorials/deploy-vyper.md]: noted

File: src/tutorials/deploy-solidity.md [hunk 2]
```diff
@@ -1,11 +1,8 @@
-# Deploying Solidity Contracts on Oasis (Compound)
+# Deploying Solidity Contracts on Oasis 
 
-[Compound Money Market](https://github.com/compound-finance/compound-money-market) is a lending market built on Ethereum. Here, it is configured for deployment on Oasis.
-This tutorial's methods for deploying Solidity contracts using Truffle and Web3.js are applicable to any Ethereum contract written in Solidity.
+This tutorial will demonstrate how to deploy Solidity contracts to the Oasis platform using Truffle and Web3.js. It should work for _any_ Solidity contract.
```
Comment (nhynes) [inline src/tutorials/deploy-solidity.md]: sentence per line. but see from the diff how it's useful?

File: src/tutorials/deploy-solidity.md [hunk 3]
```diff
@@ -134,7 +139,8 @@ truffle deploy
 
 ## Deploy Using Web3
 
-The Oasis Network also has support for [Web3.js](https://web3js.readthedocs.io/en/v1.2.0/getting-started.html), a Javascript module to help build frontends for contracts on Ethereum. This method is more involved, but useful in creating your frontend application.
+The Oasis Network also has support for [Web3.js](https://web3js.readthedocs.io/en/v1.2.0/getting-started.html), a Javascript module to help build frontends for contracts on Ethereum.
```
Comment (nhynes) [inline src/tutorials/deploy-solidity.md]: NB: we also support [ethers.js](https://github.com/adrianmcli/web3-vs-ethers). we should be consistent in advertising both alternatives.

File: src/tutorials/deploy-solidity.md [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 
-This tutorial will demonstrate how to deploy Solidity contracts to the Oasis platform using Truffle and Web3.js. It should work for _any_ Solidity contract.
+This tutorial will demonstrate how to deploy Solidity contracts to the Oasis platform using Truffle and Web3.js. 
+It should work for _any_ Solidity contract.
 For contracts written in Vyper, see the [Uniswap tutorial](./deploy-vyper.md).
 
 ## Prerequisites
 
-This tutorial assumes that you have [already installed the Oasis toolchain](https://docs.oasis.dev/quickstart.html#install-the-oasis-toolchain). Your installation was successful if the following command works without error:
+This tutorial assumes that you have [already installed the Oasis toolchain](https://docs.oasis.dev/quickstart.html#install-the-oasis-toolchain).
```
Comment (nhynes) [inline src/tutorials/deploy-solidity.md]: can you make this link relative?
```
[already installed the Oasis toolchain](/quickstart.html#install-the-oasis-toolchain)
```

File: src/tutorials/deploy-solidity.md [hunk 5]
```diff
@@ -55,61 +59,69 @@ Base HD Path:  m/44'/60'/0'/0/{account_index}
...
 
 If you want to use the Oasis Devnet, you will need to use the following URL:
 
 ```js
-provider: () => new HDWalletProvider(MNEMONIC, "https://web3.devnet.oasiscloud.io")
+provider: () => new HDWalletProvider(MNEMONIC, 'https://web3.devnet.oasiscloud.io')
 ```
 
-If it doesn't exist already, you need to include the version of the Solidity compiler, `solc`, you need for the Compound contracts, which is likely not your installed Truffle's configured default. You'll need the highest version used by the contracts, which is stated at the top (e.g.` pragma solidity ^0.4.24`). Add to `module.exports`:
+If it doesn't exist already, you need to include the version of the Solidity compiler, `solc`, you need for the Compound contracts, which is likely not your installed Truffle's configured default.
```
Comment (eauge) [inline src/tutorials/deploy-solidity.md]: I guess we are rewriting this tutorial to be something generic, so we don't need to mention Compound explicitly in this line

**Pull request closed without merging**